peer/brontide.go

```diff
@@ -461,6 +461,9 @@ type Config struct {
 	// onion messages to subscribers.
 	OnionMessageServer *subscribe.Server
 
+	// OnionMsgSender is a function that sends an onion message to any peer.
+	OnionMsgSender func([33]byte, *btcec.PublicKey, []byte) error
```

Comment:
Why do we need this vs just the existing `SendMessage` method we have? It can submit just the dest and the wire message.